webtau-junit5-examples/src/test/groovy/com/example/tests/junit5/DynamicTestsGroovyTest.groovy

```diff
@@ -4,7 +4,7 @@ import org.junit.jupiter.api.TestFactory
 
 class DynamicTestsGroovyTest {
     @TestFactory
-    def priceCalculator() {
+    def "individual tests use generated display labels"() {
```

Comment:
This will break docs

Reply:
can you point me to where it was used?

Reply:
table.md

Reply:
turns out this particular test was not used.